### PR TITLE
Parallelise remove_loops_and_sharp_turns

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/tests/test_decompose_connectivity.py [hunk 1]
```diff
@@ -28,5 +28,6 @@ def test_execution_connectivity(script_runner):
                             '--min_length', '20', '--max_length', '200',
                             '--outlier_threshold', '0.5',
                             '--loop_max_angle', '330',
-                            '--curv_qb_distance', '10')
+                            '--curv_qb_distance', '10',
+                            '--processes','1')
```
Comment (CHrlS98) [inline scripts/tests/test_decompose_connectivity.py:32]: pep8 space after comma

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/tests/test_detect_streamlines_loops.py [hunk 2]
```diff
@@ -25,5 +25,6 @@ def test_execution_filtering(script_runner):
                             in_bundle, 'bundle_4_filtered_no_loops.trk',
                             '--looping_tractogram',
                             'bundle_4_filtered_loops.trk',
-                            '-a', '270', '--qb', '--threshold', '4')
+                            '-a', '270', '--qb', '--threshold', '4',
+                            '--processes','1')
```
Comment (CHrlS98) [inline scripts/tests/test_detect_streamlines_loops.py:29]: pep8 space after comma

File: scripts/tests/test_filter_tractogram_anatomically.py [hunk 3]
```diff
@@ -27,5 +27,6 @@ def test_execution_filtering(script_runner):
     ret = script_runner.run('scil_filter_tractogram_anatomically.py',
                             in_tractogram, in_wmparc,
                             os.path.expanduser(tmp_dir.name),
-                            '--minL', '40', '--maxL', '200', '-a', '300')
+                            '--minL', '40', '--maxL', '200', '-a', '300',
+                            '--processes','1')
```
Comment (CHrlS98) [inline scripts/tests/test_filter_tractogram_anatomically.py:31]: pep8 space after comma

**Review by CHrlS98 — APPROVED** [review]
Everything works on my end.
Noticed the output of `scil_filter_tractogram_anatomically.py` is suffixed `_filtered_filtered.trk`. See line 301. I know you didn't touch this part, but should we change that?

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by AlexVCaron — APPROVED** [review]
Everything seems good on my end. All tested and it works

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**